src/otx/algorithms/detection/configs/base/data/semisl/base_semisl_is_res_data_pipeline.py

```diff
@@ -0,0 +1,140 @@
+"""Data Pipeline for Semi-Supervised Learning Detection Task."""
```

Comment:
Does this config have any other child configs than IS SSL with Resnet-50?

Reply:
No, It is better to remove base in naming